ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -5850,6 +5870,15 @@ public ShowCompactResponse showCompactions() throws HiveException {
     }
   }
 
+  public ShowCompactResponse showCompactions(String poolName) throws HiveException {
+    try {
+      return getMSC().showCompactions(poolName);
```

Comment:
i think we need to make it more generic with request object so we could filter by table/partition names as well

Reply:
That is OK, but I think it should be implemented in a separate dedicated change. It is not strictly related to priority queueing .

Reply:
agree ...